packages/nimble-components/src/accordion/specs/README.md

```diff
@@ -0,0 +1,170 @@
+                <nimble-checkbox></nimble-checkbox>
+                <span>Accordion one content</span>
+            </div>
+    </nimble-accordion-item>
+    <nimble-accordion-item>
+        <div slot="heading">Accordion two</div>
+            <div slot="default">
+                <nimble-button>Text Button</nimble-button>
+                <span>Accordion one content</span>
+            </div>
+    </nimble-accordion-item>
+    <nimble-accordion-item>
+        <div slot="heading">Accordion three</div>
+            <div slot="default">
+                <nimble-checkbox></nimble-checkbox>
+                <nimble-accordion>
+                    <nimble-accordion-item>
+                         <div slot="heading">Nested Accordion One</div>
+                            <div slot="default">
+                                <span>Nested Accordion One Content</span>
+                            </div>
+                    <nimble-accordion-item>
+                    <nimble-accordion-item>
+                         <div slot="heading">Nested Accordion Two</div>
+                            <div slot="default">
+                                <span>Nested Accordion Two Content</span>
+                            </div>
+                    <nimble-accordion-item>
+                </nimble-accordion>
+            </div>
+    </nimble-accordion-item>
+    <nimble-accordion-item>
+        <div slot="heading">Accordion four</div>
+            <div slot="default">
+                <nimble-button>Text Button</nimble-button>
+                <span>Accordion one content</span>
+            </div>
+    </nimble-accordion-item>
+</nimble-accordion>
+```
+
+### API
+
+#### Nimble Accordion
+
+[FAST accordion API](https://github.com/microsoft/fast/blob/57f3c22c6341d8a21d48b1ffb7fcbfab1ffd02d8/packages/web-components/fast-foundation/src/accordion/accordion.spec.md)
+
+-   _Component Name:_ `nimble-accordion`
+-   _Properties/Attributes:_
+    -   `appearance` - `block`, `outline`, `ghost`
+-   _Methods:_ Unchanged
+-   _Events:_ Unchanged
+-   _CSS Classes and Custom Properties that affect the component:_ Unchanged
+-   _Slots:_ Unchanged
+
+The Figma design includes appearances of the accordion header that reflect those of the `block`, `outline`, and `ghost` appearances used in other components. These designs are similar to those of the nimble-button and its `appearance` types, but the accordion does not use the `.control` css class, it uses `.region` and `.heading`. Because of this, differences in border functionality, and the possibility of excess overridden css classes if patterns are used, shared styles will not be used, and new styles for the accordion and accordion-item will be created. The appearances of `block`, `outline`, and `ghost` will be implemented under the HTML attribute `appearance`, which will use conditional css styles based on `appearance`'s value (ex. :host([appearance='outline']) will have styling for the outline button).
```

Comment:
use the code escape for code snippets
```suggestion
The Figma design includes appearances of the accordion header that reflect those of the `block`, `outline`, and `ghost` appearances used in other components. These designs are similar to those of the nimble-button and its `appearance` types, but the accordion does not use the `.control` css class, it uses `.region` and `.heading`. Because of this, differences in border functionality, and the possibility of excess overridden css classes if patterns are used, shared styles will not be used, and new styles for the accordion and accordion-item will be created. The appearances of `block`, `outline`, and `ghost` will be implemented under the HTML attribute `appearance`, which will use conditional css styles based on `appearance`'s value `(ex. :host([appearance='outline'])` will have styling for the outline button).
```